src/frontend/views/Attester/Attester.tsx

```diff
@@ -187,18 +185,18 @@ function Connect({ setSession }: { setSession: (s: Session) => void }) {
   const [processing, setProcessing] = useState(false);
   const [error, setError] = useState<'closed' | 'rejected' | 'unknown'>();
 
-  const extensions = getWindowExtensions();
+  const { kilt } = apiWindow;
+  const extensions = Object.keys(kilt);
```

Comment:
Do you even need this constant?

Reply:
I suppose not, but I thought it was nicer than writing `Object.keys(kilt)` every time